packages/strapi-plugin-users-permissions/controllers/UsersPermissions.js

```diff
@@ -160,5 +163,41 @@ module.exports = {
     } catch(error) {
       ctx.badRequest(null, [{ messages: [{ id: 'An error occurred' }] }]);
     }
+  },
+
+  getEmailTemplate: async (ctx) => {
+    ctx.send(strapi.plugins['users-permissions'].config.email);
+  },
+
+  updateEmailTemplate: async (ctx) => {
+    if (_.isEmpty(ctx.request.body)) {
+      return ctx.badRequest(null, [{ messages: [{ id: 'Cannot be empty' }] }]);
+    }
+
+    strapi.plugins['users-permissions'].config.email = ctx.request.body;
```

Comment:
This line remembers me that we are not freezing the configurations of a plugin, and we should.